src/extensions/default/InlineColorEditor/css/main.less

```diff
@@ -268,6 +280,10 @@
     vertical-align: middle;
     background: transparent;
     -webkit-font-smoothing: antialiased;
+    -moz-font-smoothing: antialiased;
+    -o-font-smoothing: antialiased;
+    -ms-font-smoothing: antialiased;
+    font-smoothing: antialiased;
```

Comment:
Is `font-smoothing` even a valid (read: non-WebKit only) property, and do all these prefixes even exist? The closest I can find is a `font-smooth` property from a very old (2002) CSS3 draft, and it has long been dropped from the spec.